sdk/storage/azdatalake/filesystem/constants.go

```diff
@@ -0,0 +1,17 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package filesystem
+
+import "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+
+// PublicAccessType defines values for AccessType - private (default) or file or filesystem.
+type PublicAccessType = azblob.PublicAccessType
+
+const (
+	File       PublicAccessType = azblob.PublicAccessTypeBlob
```

Comment:
should this be Path?

Reply:
It should be File, not path its basically anonymous access to data in a blob/file